eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubConsumer.java

```diff
@@ -51,6 +48,10 @@
     private static final AtomicReferenceFieldUpdater<EventHubConsumer, AmqpReceiveLink> RECEIVE_LINK_FIELD_UPDATER =
         AtomicReferenceFieldUpdater.newUpdater(EventHubConsumer.class, AmqpReceiveLink.class, "receiveLink");
 
+    // We don't want to dump too many credits on the link at once. It's easy enough to ask for more.
+    private static final int MINIMUM_REQUEST = 1;
+    private static final int MAXIMUM_REQUEST = 100;
```

Comment:
I'm not overly familiar with these areas in the .NET code, but 100 seems a bit low for what I think is a limit on the prefetch.   Is there any reason that you decided on such a big delta from 8000 down to just 100?

Reply:
We accept that prefetch of 8000, setting that occurs when the AmqpReceiveLink is first mapped and subscribed to. This maximum request is for after that credit of 8000 gets drained. How much more should we ask for after we've fetched those 8000? In the original code, it kept adding credits 1 by 1 and there was a note to make it less noisy.

Reply:
ahhh....  makes sense!